float64 body1
float64 body2
float64 body3
float64 body4
float64 body5
float64 body6
float64 body7
bool idleState